### ISSUE TITLE
Add lm and next sentence accuracy for run_lm_finetuning example

### ISSUE THREAD

**Issue by weiczhu** [huggingface/pytorch-pretrained-BERT#333]

**Comment by thomwolf** [comment]
Yes, feel free to submit a PR for that.
